cli/src/trusted_commands.rs

```diff
@@ -0,0 +1,309 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::{trusted_operation::perform_trusted_operation, Cli};
+use base58::{FromBase58, ToBase58};
+use codec::{Decode, Encode};
+use ita_stf::{
+	AccountId, Index, KeyPair, ShardIdentifier, TrustedCall, TrustedGetter, TrustedOperation,
+};
+use log::*;
+use my_node_runtime::Balance;
+use sp_application_crypto::{ed25519, sr25519};
+use sp_core::{crypto::Ss58Codec, sr25519 as sr25519_core, Pair};
+use sp_runtime::traits::IdentifyAccount;
+use std::path::PathBuf;
+use substrate_client_keystore::{KeystoreExt, LocalKeystore};
+
+const KEYSTORE_PATH: &str = "my_trusted_keystore";
```

Comment:
This path is duplicated - we should have one definition of the path to avoid inconsistencies.